File: codigos_exercicios/exercicio26.m
% definindo o sistema
rigid = @(t,y) [y(2) * y(3); -y(1) * y(3); -0.51 * y(1) * y(2)];

% resolvendo
options = odeset('RelTol',1e-4,'AbsTol',[1e-4 1e-4 1e-5]);
[T,Y] = ode45(rigid,[0 12],[0 1 1],options);

% plotando
plot(T,Y(:,1),'-',T,Y(:,2),'-.',T,Y(:,3),'.')

File: codigos_exercicios/python/exercicio26.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Exercicio 26
"""

import numpy as np
import scipy.integrate as integ
import matplotlib.pyplot as plt

# definindo o sistema
def rigid(y,t):
    return [y[1] * y[2], -y[0] * y[2], -0.51 * y[0] * y[1]]

t = np.arange(0,12.1,0.1)
x0 = [0, 1, 1]

x = integ.odeint(rigid,x0,t) 

plt.plot(t,x)